### DISCUSSION TITLE
Anyone seeing decorators are not enabled error?

### DISCUSSION THREAD

**Discussion by PodStuart** [nrwl/nx#27533] [Questions]
Hi there,

Is anyone seeing the following error trying to migrate to  v19.6.1, from v19.5.7 in our case. Not sure if this is a regression or not.

`SyntaxError: Decorators are not enabled.
      If you are using ["@babel/plugin-proposal-decorators", { "version": "legacy" }], make sure it comes *before* "@babel/plugin-transform-class-properties" and enable loose mode, like so:
      	["@babel/plugin-proposal-decorators", { "version": "legacy" }]
      	["@babel/plugin-transform-class-properties", { "loose": true }]`

**Comment by amin3mej** [comment]
I resolved it like this: https://github.com/nrwl/nx/issues/29010#issuecomment-2490773990
